app/src/main/java/org/mozilla/focus/urlinput/UrlInputFragment.java

```diff
@@ -94,8 +94,7 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         clearView = view.findViewById(R.id.clear);
         clearView.setOnClickListener(this);
 
-        urlAutoCompleteFilter = new UrlAutoCompleteFilter();
-        urlAutoCompleteFilter.loadDomainsInBackground(getContext().getApplicationContext());
+        urlAutoCompleteFilter = new UrlAutoCompleteFilter(getContext());
```

Comment:
the original author specify to use Application-Context here, I think we should follow the enlightenment come from the one.